test/e2e/cadvisor.go

```diff
@@ -58,7 +58,7 @@ func CheckCadvisorHealthOnAllNodes(c *client.Client, timeout time.Duration) {
 		for _, node := range nodeList.Items {
 			// cadvisor is not accessible directly unless its port (4194 by default) is exposed.
 			// Here, we access '/stats/' REST endpoint on the kubelet which polls cadvisor internally.
-			statsResource := fmt.Sprintf("api/v1beta3/proxy/nodes/%s/stats/", node.Name)
+			statsResource := fmt.Sprintf("api/v1/proxy/nodes/%s/stats/", node.Name)
```

Comment:
This is fine for now, but it's worth thinking about how we could parameterize the version, and delegate path construction to the client library (pkg/client).